src/workflow/units/execution/shell.py

```diff
@@ -24,9 +24,19 @@ def application(self):
         return {
             "name": "shell",
             "summary": "Shell Script",
-            "version": "4.2.46"
+            "version": self.version
         }
 
+    @property
+    def version(self):
+        """
+        Returns the application version used in the unit.
+
+        Returns:
+             str
+        """
+        return "4.2.46"
```

Comment:
Less important for bash/shell, but the same point as above

Reply:
Fixed.